### PR TITLE
Remove Mutual Authentication (Fixing KUKSA Server token handling)

### PR DESCRIPTION
**Pull request by erikbosch**
This is a follow up to #23. Previously we always had default client key/certs and fed them to KUKSA Server unless some other key/cert was given. So in short, they were never null or empty string. Now they might be empty unless specified in config or similar, so we must handle that scenario. But opting for removing client key support instead.

### CODE REVIEW

File: kuksa-client/kuksa_client/cli_backend/ws.py [hunk 1]
```diff
@@ -297,8 +297,6 @@ async def connect(self, _=None):
         subprotocols = ["VISSv2"]
         if not self.insecure:
             context = ssl.create_default_context()
-            context.load_cert_chain(
-                certfile=self.certificate, keyfile=self.keyfile)
```
Comment (erikbosch) [inline kuksa-client/kuksa_client/cli_backend/ws.py:301]: If we do not want to remove support for client auth then this statement must be modified by checking that both are not null before calling it

**Comment by erikbosch** [comment]
@SebastianSchildt - I noticed a side effect for ws/server connection after removing default certs/tokens, instead of adapting I think this could be a good time to remove client key/cert handling, as we anyway do not support it in Server or databroker, and do not plan to support it.

File: .github/workflows/dash.yaml [hunk 2]
```diff
@@ -35,6 +35,7 @@ jobs:
           > dependencies.txt
 
       - name: Dash license check
-        uses: eclipse-kuksa/kuksa-actions/check-dash@2
+        uses: eclipse-kuksa/kuksa-actions/check-dash@4
```
Comment (erikbosch) [inline .github/workflows/dash.yaml:38]: Trying out the dash update as well @SebastianSchildt

**Review by SebastianSchildt — APPROVED** [review]
Agree this feature can be removed. Did some manual testing, no regression (as expected) GRPC side, and als was successful using VISS
lgtm 👑

**Pull request merged**